### ISSUE TITLE
Underscores not visible in search in Ubuntu 24.04.1 LTS

### ISSUE THREAD

**Issue by justanotheranonymoususer** [microsoft/vscode#227583]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.92.2
Commit: fee1edb8d6d72a0ddff41e5f71a671c23ed924b9
Date: 2024-08-14T17:29:30.058Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Linux x64 6.8.0-40-generic snap

Steps to Reproduce:

1. Use underscores in search in the lower fields
2. See screencast

[Screencast from 2024-09-04 16-24-34.webm](https://github.com/user-attachments/assets/5b0a92fb-1f79-4fca-ac5b-b07e068c7321)

**Comment by justanotheranonymoususer** [comment]
[thx for the malware](https://www.virustotal.com/gui/file/ca741af032104ba047834838f1db94e79d21edb62bd1bbfc79e2910d5304e77d), luckily my Ubuntu doesn't run exes

**Comment by andreamah** [comment]
@daviddossett did we change the heights of input fields?

**Comment by justanotheranonymoususer** [comment]
Happens with Ctrl+P too. Happens on vscode.dev and github.dev too.

**Comment by daviddossett** [comment]
@andreamah not that I know of. Possibly a linux-specific issue.

**Comment by justanotheranonymoususer** [comment]
What the heck? what information is needed? Bad bot

**Comment by andreamah** [comment]
![Image](https://github.com/user-attachments/assets/c7bcc530-3c54-4191-b247-e8678326b08f)
Hmm, I can't seem to repro this on Ubuntu 24.04. I'm assuming that this happens in the latest version too?

If it also happens on vscode.dev and github.dev too, I suspect that it's something specific to the machine somehow. Is this like this, even when you zoom in/out?

**Comment by andreamah** [comment]
cc @TylerLeonhardt since it also happens with the quick open (ctrl+p)

**Comment by TylerLeonhardt** [comment]
This is probably reproable in any InputBox:
https://github.com/microsoft/vscode/blob/699bbcc761d4560092128fae81e97e72850a9d7b/src/vs/base/browser/ui/inputbox/inputBox.ts#L99

@justanotheranonymoususer how frequent of a VS Code user are you? Do you think this only started happening recently? If so we can try to track it down to a specific commit.

**Comment by justanotheranonymoususer** [comment]
> I'm assuming that this happens in the latest version too?

Yes, now using:

Version: 1.94.0
Commit: d78a74bcdfad14d5d3b1b782f87255d802b57511
Date: 2024-10-02T13:08:12.626Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Linux x64 6.8.0-40-generic snap

Same here:
https://vscode.dev/
https://insiders.vscode.dev/

> Is this like this, even when you zoom in/out?

Both zoom in and zoom out fix it, but in 100% it's buggy

> how frequent of a VS Code user are you?

Pretty frequent, so probably started not too long before I created the issue.
What's the easiest way for me to bisect?
It'd be nice if there would be `https://<commit>.vscode.dev/` or something...

**Comment by andreamah** [comment]
I believe that you can add `?vscode-version=COMMIT` to the end of the URL. You can also use https://github.com/microsoft/vscode-bisect to bisect the vscode version.

**Comment by TylerLeonhardt** [comment]
This is an interesting comment over in Jupyterlab: https://github.com/jupyterlab/jupyterlab/issues/16820#issuecomment-2398017679

**Comment by justanotheranonymoususer** [comment]
@TylerLeonhardt that can explain it, I updated Ubuntu fairly recently.

Maybe you just do the same thing as you did with the first field, it works fine, they should be the same anyway, shouldn't they?

![image](https://github.com/user-attachments/assets/3734f8db-08e4-4f1d-9f5c-65526fba7616)

**Comment by justanotheranonymoususer** [comment]
Other examples:

Inconsistency in settings:

![image](https://github.com/user-attachments/assets/ecbd1597-fdb5-40b5-8ac6-bcb67179eb83)

Bad profile name:

![image](https://github.com/user-attachments/assets/2be071ee-86b6-493d-9bc1-4c45f786f440)

Rename - bad:

![Screenshot from 2024-10-08 11-04-03](https://github.com/user-attachments/assets/ffded679-9e98-499e-b78d-9e8b7790e499)

After rename - OK:

![image](https://github.com/user-attachments/assets/074e719c-d0d2-4a90-8b59-3162bca1cc23)

Git commit - OK
Search extensions - OK
Keyboard shortcuts search - OK

**Comment by andreamah** [comment]
Does it get fixed if you download `fonts-ubuntu-classic`?

**Comment by justanotheranonymoususer** [comment]
@andreamah yes, that fixed it, the workaround works. Still, I'd rather not use an old version and disable updates for this package (see comments [here](https://askubuntu.com/a/1465230))

**Comment by justanotheranonymoususer** [comment]
What the hell? bad bot!

**Comment by TylerLeonhardt** [comment]
I talked to @deepak1556 about this and he mentioned that this could be an electron/chromium issue. He's going to check if there's an existing  issue there.

**Comment by kameshkotwani** [comment]
This is happening with me as well

**Comment by Melebius** [comment]
On my Ubuntu 24.04.2 LTS, I’ve noticed missing underscores on the most bottom line of the integrated terminal (might be covered by the status bar?):

![Image](https://github.com/user-attachments/assets/6b9bcca5-5cf8-4a72-a7cc-ad2253f8d424)

and also in the `Recent` list on the Welcome page (the screenshot shows the hover where underscores are displayed correctly):

![Image](https://github.com/user-attachments/assets/e2a8881c-214d-46fb-909a-f33877764442)

Might this be related?

Edit: [Installing `fonts-ubuntu-classic`](#issuecomment-2400367324) fixes the `Recent` list (and search inputs) for me but not the terminal.

**Comment by juuf** [comment]
After setting up VS code freshly in an offline environment (Red Hat 9.7), I have the same issue as @Melebius. No "_" in for the bottom line in a terminal. I have an older version of VS code there without said problem. Might be related to this issue (https://github.com/microsoft/vscode/issues/130967)?

<img width="172" height="65" alt="Image" src="https://github.com/user-attachments/assets/30341a49-4fd7-4b84-bf4e-afd33c5b3b0a" />

> 
> This bug has been fixed in the latest release of [VS Code Insiders](https://code.visualstudio.com/insiders/)!
> 
> @zzeitt, you can help us out by commenting `/verified` if things are now working as expected.
> 
> If things still don't seem right, please ensure you're on version 5f19eee5dc9588ca96192f89587b5878b7d7180d of Insiders (today's or later - you can use `Help: About` in the command palette to check), and leave a comment letting us know what isn't working as expected.
> 
> Happy Coding! 

 _Originally posted by @vscode-triage-bot in [#130967](https://github.com/microsoft/vscode/issues/130967#issuecomment-905028122)_
